### PR TITLE
Created strategy process moderation folder for documentation

### PR DESCRIPTION
**Pull request by soledadli**
This folder is to document information and discussions about how the moderators will interact with the platform:

### CODE REVIEW

File: Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Related Material
+* Moderation Workflow Diagram-December 02 2021 
+![Moderation Workflow Diagram](moderation_workflow_diagram_20211202.png)
+* [Click for detail of the workflow diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=2EoajS9YyuE2mbzpoMci&title=user_flow_autspacs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1pohsA28S36n1yIWvAJhNd5k79qazjSeg%26export%3Ddownload)
+
+## Discussed strategies are as followed:
+* Harmful reports will be sent to moderators. Administrators will have the access to view all the harmful reports, but will not receive the notifications per experience.
+* We will focus on the first workflows: **New Experiences** and **Disapproved Experiences**.
+* We will hold off the "flag for harmful reports" feature for MVP as it will complicate the current infrastructure. Apart from that, further clarity will be needed for the "harmful reports" process. 
+* Email should be avoided to use as the means of the communication in the platform as it is not user-friendly and can be quite hard for document the process.
```
Comment (GeorgiaHCA) [inline Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md:13]: hard "to" document

**Review by GeorgiaHCA — APPROVED** [review]
Looks fab! Just a minor suggestion from me. I think you've captured this really well!

**Comment by GeorgiaHCA** [comment]
Perhaps @skfantoni, @JamesSCTJ, @KirstieJane would be able to review to make sure we're all on the same page :) x x

File: Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md (hunk 1 above)
Comment (soledadli) [reply Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md]: Thanks for the check!

File: Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Meet-Up Discussion on November 25 2021
```
Comment (KirstieJane) [inline Moderation/strategy-process-moderation/20211130_summary-30-November-2021.md:1]:
```suggestion
# Meet-Up Discussion on November 25 2021

```

**Review by KirstieJane — APPROVED** [review]
This looks great - really helpful for keeping the notes together.

One thing that you could do in the future for this folder is add a document that summarises the decisions _to date_. So if a decision is changed then you change it in that file. That way you have a final description in one file while the notes from the meetings contain the history and story of how we got to it!

**Pull request merged**